iamchrissmith/uncorked
# Venue manager can associate wines

#### What does this PR do?
Allows venue managers to add a wine to their venues and to edit which venues a wine is associated with (i.e. add new ones or remove existing ones).

#### Where should the reviewer start?
`venue_wines_controller.rb`

#### How should this be manually tested?
1.`rake db:seed` if you haven't
2.`rails s`
3. log in as a manager
4. create a venue
5. visit a wine page
6. click add wine to venues
7. select venue you want
8. click save
9. click edit venues
10. uncheck/check as you like
11. click save

#### Any background context you want to provide?
@jmejia : I am not happy with the way the controller/from actions are happening. I would like to sit down and talk through a couple of points with you on how we might improve this.

#### What are the relevant tickets?
[148705547](https://www.pivotaltracker.com/story/show/148705547)

#### Screenshots (if appropriate)
#### Questions:
  - Do Migrations Need to be ran? **yes**
  - Do Environment Variables need to be set? no
  - Any other deploy steps? no
